Kickstarter-iOS/Views/Cells/PledgeExpandableHeaderRewardCell.swift

```diff
@@ -0,0 +1,111 @@
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Margin {
+    static let width: CGFloat = Styles.grid(3)
+  }
+}
+
+final class PledgeExpandableHeaderRewardCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var amountLabel: UILabel = { UILabel(frame: .zero) }()
```

Comment:
If its a single line, I think you can just do: 
```suggestion
  private lazy var amountLabel: UILabel = UILabel(frame: .zero)
```

Reply:
Ah cool, we've always just written our lazy props this way but no reason to add closures if it's not needed! Will update here 👍